apps/meeting/server.js

```diff
@@ -15,18 +15,34 @@ let protocol = 'http';
 let options = {};
 
 const chime = new AWS.Chime({ region: 'us-east-1' });
-const alternateEndpoint = process.env.ENDPOINT;
```

Comment:
We would still need this if we were to test the Chime's other (gamma) endpoint right?

Reply:
add back